ext_localconf.php

```diff
@@ -9,3 +9,31 @@
 
 $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['extbase']['commandControllers'][] =
     \NamelessCoder\AsyncReferenceIndexing\Command\ReferenceIndexCommandController::class;
+
+// Overwrite lowlevel_refindex since we want to perform the indexing and not spool into our table
```

Comment:
I would prefer simply removing this cliKey completely (or make it exit with error, referring to the command controller). However, there may be another case of this where we need a proxy as well - it is possible to trigger this from the backend module for "lowlevel" - right now, doing this fills the queue for CLI processing, but maybe it's best to allow that module to trigger the "real" processing instead... I'm still undecided about that part.